ensemble-view/src/main/java/hec/ensembleview/tabs/SingleValueSummaryTab.java

```diff
@@ -184,13 +184,16 @@ private void setActionListeners() {
         clearButton.addActionListener(e -> outputArea.setText(""));
 
         computeButton.addActionListener(e -> {
-            MetricCollectionTimeSeries value = statComputationHelper.computeTwoStepComputable(getFirstStat(), getFirstTextFieldValue(),
-                    getSecondStat(), getSecondTextFieldValue(),
-                    getSummaryType() == SingleValueSummaryType.COMPUTEACROSSENSEMBLES ||
-                            getSummaryType() == SingleValueSummaryType.COMPUTECUMULATIVE);
-            getValuesFromMetricCollectionTimeSeries(value);
-
-            tryShowingOutput(getValuesFromMetricCollectionTimeSeries(value));
+            try {
+                MetricCollectionTimeSeries value = statComputationHelper.computeTwoStepComputable(getFirstStat(), getFirstTextFieldValue(),
+                        getSecondStat(), getSecondTextFieldValue(),
+                        getSummaryType() == SingleValueSummaryType.COMPUTEACROSSENSEMBLES ||
+                                getSummaryType() == SingleValueSummaryType.COMPUTECUMULATIVE);
+                getValuesFromMetricCollectionTimeSeries(value);
+                tryShowingOutput(getValuesFromMetricCollectionTimeSeries(value));
+            } catch (Exception ex) {
+                JOptionPane.showMessageDialog(this, ex.getMessage(), "Input Error", JOptionPane.ERROR_MESSAGE);
```

Comment:
The code changes here look good. Just two very minor things that I noticed when testing the UI. (1) When entering a percentile value greater than 1, the error message is missing ...less than "OR" equal to 1...  (2) When I hit compute and the percentiles box is empty, a more useful error message might be something along the lines of "Percentile(s) less than or equal to 1 must be entered in the text field"

Reply:
Concur.  Updated the Exception messages to include "or" and added exception message to cumulative